BrickKit/bricks/src/androidTest/java/com/wayfair/brickkit/BrickFragmentTest.java

```diff
@@ -1,117 +1,53 @@
-/**
- * Copyright © 2017 Wayfair. All rights reserved.
+/*
+ * Copyright © 2017-2021 Wayfair. All rights reserved.
  */
 package com.wayfair.brickkit;
 
 import android.graphics.Color;
 import android.graphics.drawable.ColorDrawable;
-import android.os.Bundle;
 import android.os.Looper;
-import android.view.LayoutInflater;
-import android.view.View;
-import android.view.ViewGroup;
 
 import org.junit.Before;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 
-import androidx.annotation.Nullable;
-import androidx.recyclerview.widget.GridLayoutManager;
-import androidx.recyclerview.widget.OrientationHelper;
 import androidx.recyclerview.widget.RecyclerView;
-import androidx.test.InstrumentationRegistry;
 import androidx.test.runner.AndroidJUnit4;
 
 import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.assertTrue;
 import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.when;
 
 @RunWith(AndroidJUnit4.class)
 public class BrickFragmentTest {
 
-    private TestBrickFragment testBrickFragment;
-    private LayoutInflater inflater;
+    private BrickFragment testBrickFragment;
 
     @Before
     public void setup() {
         if (Looper.myLooper() == null) {
             Looper.prepare();
         }
-        testBrickFragment = new TestBrickFragment();
+        testBrickFragment = new BrickFragment() { };
```

Comment:
the empty {} is needed?

Reply:
BrickFragment is currently abstract (I can add a ticket to see if thats necessary)